src/core/configProvider/configContext.ts

```diff
@@ -87,7 +87,7 @@ export type Config = {
   htmlSanitizer: {
     allowedUnsafeTags: HtmlToReactProps['allowedUnsafeTags'];
     trustedOrigins: HtmlToReactProps['trustedOrigins'];
-    allowUnsafeSocialMediaScript: boolean;
+    trustedScriptsOrigins: string[];
```

Comment:
Do we really need a new field? Cannot we use the old "trustedOrigins" that we use for iframes? It should not make any difference whether the origin source is used for scripts or iframes -- trusted if trusted. :)